spec/support/feature_helper.rb

```diff
@@ -14,5 +14,17 @@ def sign_in(user)
     end
     expect( page ).to have_content("Signed in successfully.")
   end
+
+  def authenticated_user(options={})
+    user_options = {
+      email: "email#{rand}@fake.com",
+      password: 'password'
+    }.merge(options)
+
+    user = User.new(user_options)
+    user.skip_confirmation!
+    user.save
+    user
+  end
```

Comment:
Awesome–thanks for this! I moved it to the user factory like yours
https://github.com/jpalmieri/brewery/commit/8dcd3a2b4518b77ed2acd8844209e743ee9d7349